src/examples.md

```diff
@@ -1303,6 +1336,7 @@ function update(id)
    showElement("mobius-strip", meshing && all2 && all3 && all4);
```

Comment:
Can you increment the number of meshing miniapps above?

Reply:
Of course!